frontends/web/src/routes/account/summary/chart.tsx

```diff
@@ -529,7 +529,7 @@ class Chart extends Component<Props, State> {
                       <span className={styles.diffUnit}>%</span>
                     </span>
                   </>
-                ) : chartTotal === 0 ? null : (<Skeleton fontSize="1.125rem" minWidth="125px" />)}
```

Comment:
> Is it possible to have `chartTotal > 0` while `hasDifference` is false? Logically it can be. 🤔
Yes, this is possible, when you have a new tx in a fresh wallet and you have only one fiat value in the chart with the starting 0. Then the `hasDifference` is false.

Reply:
> Not 100% sure, but I think this Skeleton was put there because we assume that the case above is just currently in a loading state. However, I think it's fine to remove it since theoretically the above can happen (please correct me if im wrong).

at that time I think i assumed that if there is no difference and chartTotal is 0 that the app is still loading and show a skeleton. Now that isn't needed anymore. The whole component is a bit hard to read :(

Reply:
> Yes, this is possible, when you have a new tx in a fresh wallet and you have only one fiat value in the chart with the starting 0. Then the hasDifference is false.

hasDifference can also be falsy, i.e. `0` that happens when you have the base fiat currency set to BTC. In which case it can look like this:

![Screen Shot 2024-02-20 at 14 10 55](https://github.com/digitalbitbox/bitbox-wallet-app/assets/546900/7e2a5611-43c3-4ae0-a633-89fa4c7a8997)